scripts/release_binary.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+VERSION="$1"
+USER="$2"
+TOKEN="$3"
+
```

Comment:
A good practice is to 'set -e' at the beginning of the script. This will cause it to abort on the first uncatched error